.docs/actions.md

```
@@ -95,6 +95,30 @@ $grid->addAction('delete', '', 'delete!')
 	);
 ```
 
+### Using Bootstrap modal for confirmation
+
+If you'd like to display the confirmation prompt using a Bootstrap modal instead of the native `window.confirm`, simply include the following HTML snippet in your template:
+
+```html
+<div class="modal fade" id="datagridConfirmModal" tabindex="-1">
+	<div class="modal-dialog">
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)